osu.Game/Rulesets/Mods/ModDifficultyAdjust.cs

```diff
@@ -52,6 +52,20 @@ public abstract class ModDifficultyAdjust : Mod, IApplicableToDifficulty
             Value = 5,
         };
 
+        public override string SettingDescription
+        {
+            get
+            {
+                string drainRate = DrainRate.IsDefault ? "" : $"HP {DrainRate.Value}";
+                string overallDifficulty = OverallDifficulty.IsDefault ? "" : $"OD {OverallDifficulty.Value}";
+
+                string[] settings = { drainRate, overallDifficulty };
+                // filter out empty strings so we don't have orphaned commas
+                settings = Array.FindAll(settings, s => !string.IsNullOrEmpty(s));
+                return string.Join(", ", settings);
```

Comment:
```suggestion
                 return string.Join(", ", new[]
                {
                    drainRate,
                    overallDifficulty
                }.Where(s => !string.IsNullOrEmpty(s)));
```

i think this reads better.